Rethinking_2/Chp_08.ipynb

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
<p>Use _, (ax1, ax2) = plt.subplots(1, 2, figsize=[7, 4], constrained_layout=True)</p><p> </p><p>and delete fig.tight_layout()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Why, out of interest? fig.tight_layout() provides a nicer layout, I think...

Reply:
<p>Never mind... missed the <code>constrained_layout=True</code>  argument...</p>

Reply:
mostly because matplotlib people were instered in people trying it, and seeing if works as expected.